examples/CMakeLists.txt

```diff
@@ -9,25 +9,29 @@ set(EXAMPLES
 set(EXAMPLES_OPENMP threadsafe1d1 threadsafe2d2f)
 set(EXAMPLES_C guru1d1c simple1d1c simple1d1cf)
 
+find_library(MATH_LIBRARY m)
+
 foreach(EXAMPLE ${EXAMPLES})
   add_executable(${EXAMPLE} ${EXAMPLE}.cpp)
-  target_compile_features(${EXAMPLE} PRIVATE cxx_std_17)
+  target_compile_features(${EXAMPLE} PRIVATE cxx_std_14)
```

Comment:
I can build with c++11 here. Any reason to have c++14?

Reply:
At least on Mac, I encountered errors like "call to 'exp' is ambiguous with clang and cxx_std_11.

Reply:
I see, probably the example does `exp` instead of `std::exp` somewhere